### PR TITLE
Cleaning up many warnings and increasing speed of API import

### LINKED ISSUE #70: Multiple (unused) example aircraft models loaded every Aviary run
### Desired capability or behavior.

Whenever the Aviary interface is used (at any level), the api is imported - this results in every single data-file based example case being loaded. This adds additional time (anecdotally ~1 sec extra on my laptop) and can generate warnings and errors that aren't actually related to the model being ran, as well as making debugging some files significantly more difficult.

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### PR DESCRIPTION
**Pull request by johnjasa**
### Summary

I've got a bevy of smallish changes.
- changed many small causes of warnings
- avoided importing test data in the API
- changed Dymos prefix usage so things in the `timeseries` are no longer `timeseries:states` (for example) and instead they are just `timeseries` following Dymos v1.8 changes

This PR includes #94 so please review and merge that first if not already done

### Related Issues

- Resolves #70 

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/interface/default_phase_info/two_dof.py
Comment (crecine) [inline aviary/interface/default_phase_info/two_dof.py:1]: why these changes?

File: aviary/interface/methods_for_level2.py [hunk 1]
```diff
@@ -604,11 +604,9 @@ def _get_2dof_phase(self, phase_name):
         if 'cruise' not in phase_name:
             phase.add_control(
                 Dynamic.Mission.THROTTLE, targets=Dynamic.Mission.THROTTLE, units='unitless',
-                opt=False, lower=0.0, upper=1.0
```
Comment (crecine) [inline aviary/interface/methods_for_level2.py:607]: why are we removing these limits?

File: aviary/mission/gasp_based/phases/rotation_phase.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import numpy as np
+from dymos.utils.misc import _unspecified
```
Comment (crecine) [inline aviary/mission/gasp_based/phases/rotation_phase.py:2]: unused?

File: aviary/interface/methods_for_level2.py (hunk 1 above)
Comment (johnjasa) [reply aviary/interface/methods_for_level2.py:607]: not optimizer-controlled so limits don't mean nothing

File: aviary/interface/default_phase_info/two_dof.py
Comment (johnjasa) [reply aviary/interface/default_phase_info/two_dof.py:1]: removing the initial bounds? because all time is connected so the initial bounds do not matter as `input_initial` is true for each phase

File: aviary/mission/gasp_based/phases/rotation_phase.py (hunk 2 above)
Comment (johnjasa) [reply aviary/mission/gasp_based/phases/rotation_phase.py:2]: thanks, removed

**Review by jkirk5 — APPROVED** [review]

File: aviary/interface/default_phase_info/two_dof.py
Comment (crecine) [reply aviary/interface/default_phase_info/two_dof.py:1]: ahh, thanks

**Review by crecine — APPROVED** [review]
